actors.md

```diff
@@ -580,6 +579,44 @@ func SubmitPost(proofs []PoStProof, faults []FaultSet, recovered BitField, done
 	miner.NextDoneSet = done
 	miner.ArbitratedDeals.Clear()
 }
+
+func ValidateFaultSets(faults []FaultSet, recovered, done BitField) bool {
```

Comment:
nit: looks like a crossover from #305

Reply:
oh weird. yeah... let's get 305 in and i'll rebase this